lib/trie/lookup.go

```diff
@@ -0,0 +1,87 @@
+package trie
```

Comment:
Does the lib/.DS_Store need to be in this PR?

Reply:
removed!

Reply:
add license pls

Reply:
^ poke on the above